puppet/modules/jenkins_job_builder/files/config/scripts/gemset_cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ruby=2.0.0
```

Comment:
Yeah, just setting 'ruby' if it's unset would work.  I only see a difference in using `@gemset`, which could be safely removed.